CollegeRecords/src/main/resources/config.xml

```diff
@@ -3,6 +3,13 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
 
+    <bean class="dao.DbProps">
+        <property name="driver" value="com.mysql.cj.jdbc.Driver" />
+        <property name="url" value="jdbc:mysql://localhost:3306/vali_schema" />
+        <property name="user" value="vali" />
+        <property name="password" value="123456" />
+    </bean>
```

Comment:
try to put here as a bean the data source itself, not just a container for its properties